js/cart.js

```diff
@@ -19,23 +19,45 @@ function renderCart() {
 }
 
 // TODO: Remove all of the rows (tr) in the cart table (tbody)
-function clearCart() {}
+function clearCart() {
+  document.querySelector('#cart tbody').innerHTML = '';
+}
+  
+
 
 // TODO: Fill in the <tr>'s under the <tbody> for each item in the cart
 function showCart() {
-
   // TODO: Find the table body
-
+  let tableBody =  document.querySelector('#cart tbody');
   // TODO: Iterate over the items in the cart
-  // TODO: Create a TR
+  for (i = 0; i < cart.length; i++) {
+    // TODO: Create a TR
+  let tableRows = document.createElement('tr');
+  tableRows.textContent = "Product";
+  tableBody.appendChild(tableRows);
+
   // TODO: Create a TD for the delete link, quantity,  and the item
-  // TODO: Add the TR to the TBODY and each of the TD's to the TR
+  let tableDataDelete = document.createElement('td');
+  tableDataDelete.textContent = 'Delete Link';
+  tableRows.appendChild(tableDataDelete);
 
+  let tableDataQuantity = document.createElement('td');
+  tableDataQuantity.textContent = 'Quantity';
+  tableRows.appendChild(tableDataQuantity);
+
+  let tableDataItem = document.createElement('td');
+  tableDataItem.textContent = 'Item';
+  tableRows.appendChild(tableDataItem);
+  }
+  // TODO: Add the TR to the TBODY and each of the TD's to the TR
 }
 
-function removeItemFromCart(event) {
 
+button.addEventListener('click', removeItemFromCart);
```

Comment:
Don't forget to assign your variables properly! This line's giving you an error.